pkg/cmd/init/init.go

```diff
@@ -107,7 +123,7 @@ func NewCmd(f *cmdutil.Factory) *cobra.Command {
 }
 
 func (cmd *InitCmd) run(info *InitInfo, options *contracts.AzionApplicationOptions, c *cobra.Command) error {
-	logger.Debug("Running init subcommand from edge_applications command tree")
+	logger.Debug("Running init command")
 
 	path, err := cmd.GetWorkDir()
```

Comment:
what do you think about initializing this command globally outside the scope of the command so that it's accessible throughout the system?